addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss

```diff
@@ -0,0 +1,25 @@
+.o_newsletter_modal {
+    header {
+        padding: 0;
+    }
```

Comment:
Still needed ? The header is gone, no ?

Reply:
We are not removing header as we need to add close button in it.  We could add a close button in the `modal-body` but with it, things got a bit complicated. We have to create close button element manually, even empty `oe_structure` will not display "DRAG BUILDING BLOCKS HERE" message as `model-body` has a close button inside `oe_structure` element. to fix this we need to wrap it to the extra div so on and so on and the chain reaction is started :p

Reply:
if we are not using this `renderHeader` option may be we can remove it from the Dialog class.
what do you think?

Reply:
It's ok like that, thanks :+1: